dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/support/AbstractRegistryTest.java

```diff
@@ -36,6 +40,46 @@
     private AbstractRegistry abstractRegistry;
     private boolean notifySuccess;
 
+    private static final Map<String,String> parametersProvider=new LinkedHashMap<>();
+    private static final Map<String,String> parametersConsumer=new LinkedHashMap<>();
+
+    private URL mockUrl = new URL("dubbo", "127.0.0.0", 2200);
+    static{
+        parametersProvider.put("anyhost","true");
+        parametersProvider.put("application","demo-provider");
+        parametersProvider.put("dubbo","2.0.2");
+        parametersProvider.put("generic","false");
+        parametersProvider.put("interface","org.apache.dubbo.demo.DemoService");
+        parametersProvider.put("methods","sayHello");
+        parametersProvider.put("pid","1489");
+        parametersProvider.put("side","provider");
+        parametersProvider.put("timestamp",String.valueOf(System.currentTimeMillis()));
+
+        parametersConsumer.put("application", "demo-consumer");
+        parametersConsumer.put("category", "consumer");
+        parametersConsumer.put("check", "false");
+        parametersConsumer.put("dubbo", "2.0.2");
+        parametersConsumer.put("interface", "org.apache.dubbo.demo.DemoService");
+        parametersConsumer.put("methods", "sayHello");
+        parametersConsumer.put("pid", "1676");
+        parametersConsumer.put("qos.port", "333333");
+        parametersConsumer.put("side", "consumer");
+        parametersConsumer.put("timestamp", String.valueOf(System.currentTimeMillis()));
+
+
+    }
+
+    private List<URL> getList() {
+        List<URL> list = new ArrayList<>();
+        URL url_1 = new URL("dubbo", "127.0.0.0", 1000);
```

Comment:
don't use underscore `_`, instead, it's fine to name them as `url1`, `url2`, etc.